File: CC102/Game.java
import java.util.Scanner;

public class Game {
    public static void main (String [] args) {
        try(Scanner input = new Scanner(System.in)) {
            System.out.println("Guess the number from 1-10!");
            System.out.println();
            System.out.println("Mechanics:");
            System.out.println("1. You only have 3 attempts.");
            System.out.println("2. Enter a number from 1-10.");
            System.out.println("3. Enter the number 0 to exit.");
            System.out.println("4. Enjoy.");
            System.out.println();

            for (int i=1;i<=3;i++) {
                System.out.print("Your Guess? ");
                int g = input.nextInt();

                if (g == 5) {
                    System.out.println("Congratulations! You've Guessed the Right Number!");
                    break;
                } else if (g == 0) {
                    System.out.println("Goodbye!");
                    break;
                } else {
                    System.out.println("Wrong!");
                    System.out.println();
                }
            }
        }
    }
}

File: CC104/Game.py

import random

def guessing_game():
    number_to_guess = random.randint(1, 100)
    attempts = 0

    print("Welcome to the Number Guessing Game!")
    print("I'm thinking of a number between 1 and 100. Can you guess what it is?")

    while True:
        guess = int(input("Enter your guess: "))
        attempts += 1

        if guess < number_to_guess:
            print("Too low!")
        elif guess > number_to_guess:
            print("Too high!")
        else:
            print(f"Congratulations! You guessed it in {attempts} attempts.")
            break

    if attempts >= 10:
        print(f"Sorry, you've run out of guesses. The number was {number_to_guess}.")

guessing_game()
